src/modules/reading-room/components/DurationInput/DurationInput.tsx

```diff
@@ -1,8 +1,15 @@
-import { TextInput as Base, TextInputProps } from '@meemoo/react-components';
+import { TextInput, TextInputProps } from '@meemoo/react-components';
 import { FC } from 'react';
 
+import { durationRegex } from '@reading-room/components/DurationInput/DurationInput.consts';
+
+import style from './DurationInput.module.scss';
+
 const DurationInput: FC<TextInputProps> = (props) => (
-	<Base {...props} type="time" pattern="([01][0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9])" step="1" />
+	<div className={style['c-duration-input']}>
+		<TextInput {...props} type="text" pattern={durationRegex} step="1" />
+		<span className={style['c-duration-input--placeholder']}>uu:mm:ss</span>
```

Comment:
Waarom kan deze niet via `placeholder` worden doorgegeven aan `TextInput`?

Reply:
omdat de placeholder van een textinput getoond word links in het textveld, maar deze "placeholder" wil ik rechts tonen.
En het is niet echt een placeholder, het is meer een indicator die altijd zichtbaar blijft.
Ik kan gerust de naam van de klasse aanpassen naar "c-duration-input--format-indicator" of zoiets als dat duidelijker is.